src/components/QuranReader/ReadingView/Line.tsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 import Word from 'types/Word';
 import VerseText from 'src/components/Verse/VerseText';
+import classNames from 'classnames';
 import styles from './Line.module.scss';
 
 type LineProps = {
   words: Word[];
   lineKey: string;
+  hasSpecialLayout: boolean;
 };
 
-const Line = ({ lineKey, words }: LineProps) => (
-  <div id={lineKey} className={styles.container}>
-    <div className={styles.line}>
+const Line = ({ lineKey, words, hasSpecialLayout }: LineProps) => (
+  <div id={lineKey} className={classNames(styles.container, { [styles.inline]: hasSpecialLayout })}>
```

Comment:
Nothing actionable for this PR: curios how div vs spans perform for SEO. I'd guess that search engines treat divs as "separate" content. We can investigate it later iA

Reply:
Good question, we can look into it later inshaAlla